docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -3110,6 +3120,58 @@ definitions:
         example: 'data.security.team'
 
 
+  Annotations:
+    type: object
+    description: >
+      # [Work in progress] Annotations of the Nakadi resource.
+
+      A Nakadi event-type or a subscription is considered as a Nakadi resource.
+      A Nakadi resource can have a set of annotations associated with it.
+      They follow the same rules of annotations in Kubernetes resources.
```

Comment:
can we avoid using Kubernetes in Nakadi API doc? in general there is no direct correlation between K8s and Nakadi annotations even though they can have a similar or the same set of rules.

Reply:
Good point.